### DISCUSSION TITLE
spell error in api key there is showing ape we need to change that

### DISCUSSION THREAD

**Discussion by Hitakshu-Goswami** [monkeytypegame/monkeytype#6080] [Polls]
anyone else showing ape please let us know so that we can fix that instead of ape we can fix as api 
hope this find as needful

**Comment by fehmer** [comment]
Api key are intentionally named `ape keys`.

**Reply by basola21** [reply]
is there a reason for this ?

**Reply by byseif21** [reply]
check the site name again ?

**Reply by Miodec** [reply]
Monkeys, Apes, API.
